rest/server_context.go

```diff
@@ -210,7 +210,7 @@ func (sc *ServerContext) AllDatabases() map[string]*db.DatabaseContext {
 func (sc *ServerContext) numIndexWriters() (numIndexWriters, numIndexNonWriters int) {
 
 	for _, dbContext := range sc.databases_ {
-		if dbContext.BucketSpec.FeedType != base.DcpShardFeedType {
+		if strings.ToLower(dbContext.BucketSpec.FeedType) != base.DcpShardFeedType {
```

Comment:
Seems like FeedType shouldn't be a string in BucketSpec - we should do this kind of strings.ToLower processing once when we parse the config, and then store as an enum/custom type on the BucketSpec.  Otherwise it feels like we're going to forget this again in future.

Could be spun off as a separate ticket, but since you're in the middle of the FeedType code in this PR, maybe it's a reasonable time to make the change.